src/components/FoundationsMenu.js

```diff
@@ -1,42 +1,122 @@
-import FoundationsImage from "../components/FoundationsImage"
-import customAxios from "../axios"
-import { useState, useEffect } from "react"
+import FoundationsImage from "../components/FoundationsImage";
+import customAxios from "../axios";
+import { useState, useEffect } from "react";
+import NotFound from "../pages/NotFound";
+
+function NextPage(
+  route,
+  page,
+  setPage,
+  setFoundations,
+  setDisableNext,
+  setDisablePrev
+) {
+  customAxios
+    .get(route + (page + 1))
+    .then((response) => {
+      setFoundations(response.data);
+      if (response.data.length < 5) {
+        setDisableNext(true);
+      }
+    })
+    .catch((error) => setFoundations(null));
+  setPage(page + 1);
+  setDisablePrev(false);
+}
+
+function PreviousPage(
+  route,
+  page,
+  setPage,
+  setFoundations,
+  setDisableNext,
+  setDisablePrev
+) {
+  customAxios
+    .get(route + (page - 1))
+    .then((response) => setFoundations(response.data))
+    .catch((error) => setFoundations(null));
+  if (page - 1 === 1) setDisablePrev(true);
+  setPage(page - 1);
+  setDisableNext(false);
+}
 
 const FoundationsMenu = () => {
-  const [foundations, setFoundations] = useState([])
+  const [foundations, setFoundations] = useState([]);
+  const [disablePrev, setDisablePrev] = useState(true);
+  const [disableNext, setDisableNext] = useState(false);
+  const [page, setPage] = useState(1);
+  const [route, setRoute] = useState(
```

Comment:
why are you using this hook? baseURL isn't define in axios config?

Reply:
you mean the route variable ? I think you are right, I don't need to use that hook

Reply:
Yes, this project use axios configuration to declare baseURL one time in the project. The only thing you need to add in the path

Reply:
Please Fix this in all uploaded documents in this PR